tools/DC_api_wrappers/DC_wrappers.py

```diff
@@ -0,0 +1,224 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Wrappers for fetching schema from DC API.
+"""
+
+import ast
+import copy
+import json
+import logging
+import os
+import sys
+from absl import app
+from absl import flags
+from sys import path
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string('dcid', None, 'dcid of the node to query')
+flags.DEFINE_string('dc_output_path', './prefetched_outputs/',
+                    'Path to store the output')
+flags.DEFINE_boolean('force_fetch', False,
+                     'forces api query and not return cached result')
+
+_MODULE_DIR = os.path.dirname(os.path.realpath(__file__))
+path.insert(1, os.path.join(_MODULE_DIR, '..'))
+
+from download_utils.requests_wrappers import request_post_json
+
+# logging.basicConfig() # you need to initialize logging, otherwise you will not see anything from requests
+# logging.getLogger().setLevel(logging.DEBUG)
+# requests_log = logging.getLogger("urllib3")
+# requests_log.setLevel(logging.DEBUG)
+# HTTPConnection.debuglevel = 1
+# requests_log.propagate = True
+
+
+def dc_check_existence(dcid_list: list,
+                       use_autopush: bool = True,
+                       max_items: int = 450) -> dict:
+    """Checks if a given list of dcids are present in DC.
+
+    Args:
+        dcid_list: List of dcids to be queried for existence.
+        use_autopush: Boolean value to use autopush API and not public API.
+        max_items: Limit of items to be queried in a single POST request.
+
+
+    Returns:
+        Dict object with dcids as key values and boolean values signifying existence as values.
```

Comment:
Please highlight how it is different from the `datacommons` module, for instance:
- makes REST API calls with retries on timeout.

Reply:
fixed.